### DISCUSSION TITLE
KDL vs SDL

### DISCUSSION THREAD

**Discussion by casey** [kdl-org/kdl#280] [Q&A] [answered]
I read that KDL is derived from SDL, so I'm curious how KDL and SDL differ. Thanks!

**Comment by tabatkins** [comment, marked as answer]
Answered in [the FAQ](https://github.com/kdl-org/kdl#ok-then-why-not-sdlang) - please let us know if there are details missing that you'd like the answer to!

**Reply by casey** [reply]
Oh nice, thank you, I missed that 😂
